docs/configuration/external-databases.md

```diff
@@ -0,0 +1,138 @@
+# Using external databases
+
+
+You can use an external database for the cloud controller and uaa by providing following values:
+
+```yaml
+#@data/values
+---
+capi:
+  database:
+    adapter: postgres
+    host: <host>
+    port: <port>
+    user: <user>
+    password: <password>
+    name: <database>
+
+uaa:
+  database:
+    adapter: postgresql
+    host: <host>
+    port: <port>
+    user: <user>
+    password: <password>
+    name: <database>
+    ca_cert: <ca certificate for tls>
+```
+
+
+Please note to the **different spelling** of `postgres`  and `postgresql` for capi and uaa.
+
+
+## Limitations
+
+The traffic to an external database will not be encrypted. This will be changed in the near future.
+
+In case of uaa, tls can be enabled by providing `ca_cert`.
+
+## Setup
+
+As prerequisite, you need to execute the following steps to configure your postgres installation for cf-for-k8s:
+
+1. Set the environment variables
+
+    ```bash
+    export VALUES_FILE=<path to file from above>
+    export PGPASSWORD=<password of postgres super user>
+    export PGHOST=<host where postres is running>
+    ```
+
+2. Run the following script. It will
+   * create one database each for Cloud Controller and UAA
+   * create one user each for Cloud Controller and UAA
+   * activate the `citext` extension for each of these databases
+
+    ```bash
+    VALUES_JSON=$(yaml2json "$VALUES_FILE")
```

Comment:
There doesn't seem to be a clear utility with the name of `yaml2json`. I found a few different projects, but was unsure which one was used. Can we remove the dependency on `yaml2json`?

Alternatives:
* Use [yq](https://github.com/mikefarah/yq).
* Have the shell script export values and interpolate them into a heredoc that creates the db-values.yaml.

Reply:
I will change it to yq